modules/shared/src/main/scala/io/constellationnetwork/security/Hash.scala

```diff
@@ -25,11 +24,19 @@ object hash {
 
   object Hash {
 
-    def hashCodeFromBytes(bytes: Array[Byte]): HashCode =
-      Hashing.sha256().hashBytes(bytes)
-
-    def fromBytes(bytes: Array[Byte]): Hash =
-      Hash(hashCodeFromBytes(bytes).toString)
+    def sha256FromBytes(bytes: Array[Byte]): Array[Byte] = {
+      val md = MessageDigest.getInstance("SHA-256")
+      md.update(bytes)
```

Comment:
Is it performant to create a new MessageDigest for each run? afaik google uses clone before each use, to use a single instance but keep it immutable/"clean" for external use.

can you run some benchmark to compare the old and new code? as I wrote in a separate comment, hashing is the most base level of the codebase and node runs it very very often.

Reply:
Ok, so what do we gain removing guava?